demo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-demo",
-  "version": "7.0.0",
```

Comment:
Do we care though? This is still private

Reply:
No, I can undo it.